### PR TITLE
Clarify support for OpenTelemetry custom events (Publish on 11/18/2022)

### PR DESCRIPTION
**Pull request by rhetoric101**
This PR has fallen through the cracks. Let's get this reviewed and published on 11/18/2022.

### CODE REVIEW

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx [hunk 1]
```diff
@@ -1061,9 +1061,9 @@ Conditions, violations, incidents, and issues all work for any entity.
     id="cust-evnts"
     title="Custom events"
   >
-    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic: ❌ 
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry in New Relic 🟡
 
-    OpenTelemetry users can report custom events by using New Relic’s [Event API](/docs/data-apis/ingest-apis/introduction-event-api/).
+    You can report custom events from OpenTelemetry by using the New Relic [Event API](/docs/data-apis/ingest-apis/introduction-event-api/). Alternatively, you can report on custom events using the OpenTelemetry events API via OTLP. These events are currently stored and queried as New Relic logs.
```
Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:1066]: consider adding link to logs section of data types doc where we mention 'New Relic logs'. https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#log-data

File: src/content/docs/more-integrations/open-source-telemetry-integrations/jenkins/monitoring-jenkins-ot.mdx [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
   - Open source telemetry integrations
   - OpenTelemetry
   - Jenkins
   - Quickstart
 metaDescription: Here is a simple example to setup the Jenkins OpenTelemetry plugin to send data to New Relic.
+redirects:
+  - /docs/more-integrations/open-source-telemetry-integrations/opentelemetry/jenkins/monitoring-jenkins-ot
 ---
 
 import opentelemetryJenkis01 from 'images/opentelemetry_screenshot-full_jenkis-01.png'
```
Comment (zuluecho9) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/jenkins/monitoring-jenkins-ot.mdx:14]: i know you didn't do this but if you felt like fixing. a minor typo in there for image: "opentelemetryJenkis01"

**Review by zuluecho9 — DISMISSED** [review]

File: src/content/docs/more-integrations/open-source-telemetry-integrations/jenkins/monitoring-jenkins-ot.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/jenkins/monitoring-jenkins-ot.mdx:14]: @zuluecho9 I fixed that typo for this Jenkins image.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx:1066]: @zuluecho9 I added a link to data types:

These events are currently stored and queried as New Relic logs. For more about logs, see [New Relic data types](/docs/data-apis/understand-data/new-relic-data-types/#log-data).

**Comment by rhetoric101** [comment]
Closing temporarily to trigger a new build.

**Pull request merged**